static/app/views/performance/transactionDetails/content.tsx

```diff
@@ -135,6 +135,8 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
       organization_id: parseInt(organization.id, 10),
       project_platforms: getSelectedProjectPlatforms(location, projects),
       has_otel: event?.contexts?.otel !== undefined,
+      sdk_name: event?.sdk?.name,
```

Comment:
@AbhiPrasad Oh boy...we should not be calling `trackAnalyticsEvent` in a render method. Also, can you switch to using `trackAdvancedAnalyticsEvent` which is the newer style analytics? See: https://develop.sentry.dev/analytics/#for-frontend-events

Also, you can likely use `getAnalyticsDataForEvent` to do a lot of heavy lifting on the data you pass in.